src/components/Select/styles.ts

```diff
@@ -1,5 +1,5 @@
 import { css } from '@emotion/react';
 
 export const select = css`
-  min-width: 200px;
+  width: 420px;
```

Comment:
Will it work on smaller devices?

Reply:
I added styles for devices with extension less than 480px